kentikapi/internal/resources/base.go

```diff
@@ -126,3 +134,13 @@ func (b BaseAPI) DeleteAndValidate(ctx context.Context, url string, output inter
 
 	return nil
 }
+
+func (b BaseAPI) logPayload(msg, url string, payload interface{}) {
+	if b.LogPayloads {
+		if payload == "" {
+			log.Printf("%s %s", msg, url)
```

Comment:
```suggestion
			log.Printf("%s %s\n", msg, url)
```
Isn't new line character necessary?

Reply:
Checked - not necessary, looks like log.Printf automatically ending with \n